modules/dnn/src/layers/normalize_bbox_layer.cpp

```diff
@@ -65,16 +65,12 @@ class NormalizeBBoxLayerImpl CV_FINAL : public NormalizeBBoxLayer
     {
         if (backendId == DNN_BACKEND_INFERENCE_ENGINE)
         {
-            if (pnorm != 2)
+            if (!(pnorm == 2 || pnorm == 1)) {
```

Comment:
`pnorm` is not used for IE layer creation so I think we still need to allow only `pnorm == 2` for now.